### DISCUSSION TITLE
Intune

### DISCUSSION THREAD

**Discussion by narendrapuranik** [community/community#130700] [Programming Help]
### Select Topic Area

Question

### Body

How to set password protection to install any app on windows system, only system admin can enter password to install the app, but already installed programs not get affected

**Comment by HemalathaT2** [comment]
Go to Settings > Update & Security > For developers.
Under Developer settings, toggle the switch Install apps from unknown sources to the Off position.
Click Yes to confirm the change.

**Comment by JamesConlan96** [comment]
The policies outlined at [https://learn.microsoft.com/en-us/windows/security/application-security/application-control/user-account-control/settings-and-configuration?tabs=intune#user-account-control-configuration](https://learn.microsoft.com/en-us/windows/security/application-security/application-control/user-account-control/settings-and-configuration?tabs=intune#user-account-control-configuration) may help you.
